lib/tasks/TaskUtil.js

```diff
@@ -108,6 +108,28 @@ class TaskUtil {
 		return this._projectBuildContext.isRootProject();
 	}
 
+	/**
+	 * Retrieves all build options. If not a single option is stored, an empty object is returned.
+	 *
+	 * @returns {object} The build options
+	 * @public
+	 */
+	getBuildOptions() {
+		return this._projectBuildContext.getOptions();
+	}
+
+	/**
+	 * Retrieves a build option defined by it's <code>key</code.
+	 * If no option with the given <code>key</code> is stored, <code>undefined</code> is returned.
+	 *
+	 * @param {string} key The option key
+	 * @returns {object} The build options
```

Comment:
@returns {object|undefined} The build option (or undefined)

It's a single option here, not options.

Reply:
Thanks @codeworrior. Done